Packs/CommonPlaybooks/Playbooks/playbook-Convert_file_hash_to_corresponding_hashes.yml

```diff
@@ -369,25 +381,386 @@ tasks:
     view: |-
       {
         "position": {
-          "x": 970,
-          "y": 110
+          "x": 1080,
+          "y": 60
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "39":
+    id: "39"
+    taskid: 336d7e69-334b-490c-81c4-4b48788a56be
+    type: condition
+    task:
+      id: 336d7e69-334b-490c-81c4-4b48788a56be
+      version: -1
+      name: Is the hashes retrieved?
+      type: condition
+      iscommand: false
+      brand: ""
+      description: ""
+    nexttasks:
+      '#default#':
+      - "18"
+      "yes":
+      - "40"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
```

Comment:
What Happens if the value is not SHA256? I would check the length of the value is 64